packages/atomic/src/components/commerce/atomic-commerce-pager/atomic-commerce-pager.ts

```diff
@@ -123,66 +137,95 @@ export class AtomicCommercePager
-          },
-        })}
-        ${renderPageButtons({
-          props: {
-            i18n: this.bindings.i18n,
-          },
-        })(
-          html`${pagesRange.map((pageNumber, index) =>
-            keyed(
-              `page-${pageNumber}-${index}`,
-              renderPagerPageButton({
-                props: {
-                  isSelected: pageNumber === this.pagerState.page,
-                  ariaLabel: this.bindings.i18n.t('page-number', {
-                    pageNumber: pageNumber + 1,
-                  }),
-                  onChecked: async () => {
-                    this.pager.selectPage(pageNumber);
-                    await this.focusOnFirstResultAndScrollToTop();
-                  },
-                  page: pageNumber,
-                  groupName: this.radioGroupName,
-                  text: (pageNumber + 1).toLocaleString(
-                    this.bindings.i18n.language
-                  ),
+          })(html`
+            ${renderPagerPreviousButton({
+              props: {
+                icon: this.previousButtonIcon,
+                disabled: this.pagerState.page === 0,
+                i18n: this.bindings.i18n,
+                onClick: async () => {
+                  this.pager.previousPage();
+                  await this.focusOnFirstResultAndScrollToTop();
                 },
-              })
-            )
-          )}`
-        )}
-        ${renderPagerNextButton({
-          props: {
-            icon: this.nextButtonIcon,
-            disabled: this.pagerState.page + 1 >= this.pagerState.totalPages,
-            i18n: this.bindings.i18n,
-            onClick: async () => {
-              this.pager.nextPage();
-              await this.focusOnFirstResultAndScrollToTop();
-            },
-          },
-        })}
-      `)}`
-    )}`;
+                ref: (el) => this.previousButton.setTarget(el as HTMLElement),
+              },
+            })}
+            ${renderPageButtons({
+              props: {
+                i18n: this.bindings.i18n,
+              },
+            })(html`
+              ${pagesRange.map((pageNumber, index) =>
+                keyed(
+                  `page-${pageNumber}-${index}`,
+                  renderPagerPageButton({
+                    props: {
+                      isSelected: pageNumber === this.pagerState.page,
+                      ariaLabel: this.bindings.i18n.t('page-number', {
+                        pageNumber: pageNumber + 1,
+                      }),
+                      onChecked: async () => {
+                        this.pager.selectPage(pageNumber);
+                        await this.focusOnFirstResultAndScrollToTop();
+                      },
+                      page: pageNumber,
+                      groupName: this.radioGroupName,
+                      text: (pageNumber + 1).toLocaleString(
+                        this.bindings.i18n.language
+                      ),
+                      onFocusCallback: async (
```

Comment:
This is quite a lenghty callback!

Could we declare it as a private method and bind it here?